fluent-bit-to-loki/pkg/lokiplugin/utils.go

```diff
@@ -87,6 +88,31 @@ func autoLabels(records map[string]interface{}, kuberneteslbs model.LabelSet) er
 	return nil
 }
 
+var tagKey = "tag"
```

Comment:
I am not sure. What other values can be set? If they are exposed, they also need to be properly validated, otherwise the plugin can be easily misconfigured.

Reply:
For example `tagKey` is configuration which comes from fluent-bit Configmap, and it will depend on the value we use there

Reply:
I could not see where `tagKey` is configured in https://github.com/gardener/gardener/blob/master/charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml.
But I do see `tagPrefix` there.

Reply:
This change is still not on g/g but should look like this:

1) New lua script will be added:

```
inset_tag_into_record.lua: |-
    function insertTagIntoRecord(tag, timestamp, record)
      record["tag"] = tag

      return 1, timestamp, record
    end
```

2) New Filter will be created which will use this script
```
    [FILTER]
        Name                lua
        Match               kubernetes.*
        script                inset_tag_into_record.lua
        call                   insertTagIntoRecord
```

Reply:
@vpnachev this PR is a draft. We will expose those variables.

Reply:
:rofl:

Reply:
Now it is really a `draft` PR.